interop-scoring/category-data.json

```
@@ -333,5 +333,258 @@
         ]
       }
     ]
+  },
+  "2024": {
+    "categories": [
+      {
+        "name": "interop-2024-accessibility",
+        "labels": [
+          "interop-2024-accessibility"
+        ]
+      },
+      {
+        "name": "interop-2024-starting-style-transition",
```

Oh, I thought two labels was clever to avoid picking a single name, but you had to anyway. I suggest making it painfully long:

```suggestion
        "name": "interop-2024-starting-style-transition-behavior",
```